etc/scripts/renew_certs_and_redeploy.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Assuming local.env has BENTO_PROJECT_DIR and BENTO_USER_GROUP set
+source ../../local.env
```

Comment:
if this script isn't called from its own directory, this path will break - maybe use one of the techniques from https://stackoverflow.com/questions/59895/how-do-i-get-the-directory-where-a-bash-script-is-located-from-within-the-script ?